drivers/wifi/siwx91x/siwx91x_wifi.c

```diff
@@ -967,6 +973,144 @@ static int siwx91x_dev_init(const struct device *dev)
 	return 0;
 }
 
+static int siwx91x_convert_z_sl_twt_req_type(enum wifi_twt_setup_cmd z_req_cmd)
+{
+	switch (z_req_cmd) {
+	case WIFI_TWT_SETUP_CMD_REQUEST:
+		return REQUEST_TWT;
+	case WIFI_TWT_SETUP_CMD_SUGGEST:
+		return SUGGEST_TWT;
+	case WIFI_TWT_SETUP_CMD_DEMAND:
+		return DEMAND_TWT;
+	default:
+		return -EINVAL;
+	}
+}
+
+static int siwx91x_set_twt_setup(struct wifi_twt_params *params, sl_wifi_twt_request_t *twt_req)
+{
+	sl_status_t status = SL_STATUS_OK;
```

Comment:
You don't have to initialize this variable since you are going to overwrite it anyway (rationale: the reader will try to look for where this value is used).